change/@ni-nimble-components-aca919ed-c567-4f7b-8c65-7310408a45b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Use anchored region in Select and Combobox",
```

Comment:
When this goes in, what follow-up work will be necessary to fix the dropdown clipping behavior in SLE slide outs? Are there workarounds we need to remove or is it sufficient to simply uptake the new nimble-angular package?

Just want to make sure that work, if any, doesn't get lost (either you plan to do it as a follow up or it gets tracked in AzDO work items).

Reply:
We should revert the mitigation I introduced to the `sl-properties` dropdowns in https://ni.visualstudio.com/DevCentral/_git/Skyline/pullrequest/320276. I will do it as a follow-up.